scalableminds/webknossos
# Fix two bugs related to transferring task instances

- Fixed a bug where the task search showed duplicates if a user had multiple instances of a task (as made possible by the transfer functionality) – this never worked before, it was just such a rare scenario that we never noticed this.

- Fixed a bug where showing active users of a project, and transferring their tasks was broken. – this was a regression introduced in #5334 

### URL of deployed dev instance (used for testing):
- https://fixtransfertasks.webknossos.xyz

### Steps to test:
- Create task with multiple instances, request an instance for yourself
- Sign up second user, assign matching experience, get another instance of said task as this user
- as the admin, click “show active users” in project list, transfer tasks to yourself. you have two open instances now
- in task search, search by user (yourself), result should not be duplicated

### Issues:
- fixes #5442

------
